test/test_custom_ops.py

```diff
@@ -1819,10 +1819,16 @@ def op_with_incorrect_schema(testcase, name):
 class MiniOpTest(CustomOpTestCaseBase):
     test_ns = "mini_op_test"
 
-    def _op_delayed_backward_error(self, name):
+    def _op_delayed_backward_error(self):
+        name = "delayed_error"
+        qualname = f"{self.test_ns}::{name}"
+        try:
+            return torch._library.utils.lookup_op(qualname)
+        except AttributeError:
+            pass
```

Comment:
hmm?

Reply:
Fixed, I was being lazy. We needed a way to initialize the operator once